modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Instructions.kt

```diff
@@ -140,6 +167,9 @@ object Instructions {
 
     fun removePublicKey(accountId: AccountId, pubKey: PublicKey) = burnPublicKey(accountId, pubKey)
 
+    /**
+     * Instruction for granting `can_set_key_value_in_user_assets` permission to an account
```

Comment:
I think better will be used link to the permission here and below
```suggestion
     * Instruction for granting [CAN_SET_KEY_VALUE_USER_ASSETS_TOKEN] permission to an account
```